plugins.json

```diff
@@ -4193,9 +5053,269 @@
       }
     },
     "enterprise": false,
+    "platforms": {},
     "updated_at": "2020-05-28T19:23:38.781696714Z"
   },
   {
+    "repo_name": "mattermost-plugin-zoom",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-zoom",
+    "icon_data": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOTEyIiBoZWlnaHQ9IjkxMiIgdmlld0JveD0iMCAwIDkxMiA5MTIiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxjaXJjbGUgY3g9IjQ1NiIgY3k9IjQ1NiIgcj0iNDU2IiBmaWxsPSIjMkQ4Q0ZGIi8+CjxwYXRoIGQ9Ik0xOTkuNSAzNDkuMTI1QzE5OS41IDMzMy4zODUgMjEyLjI2IDMyMC42MjUgMjI4IDMyMC42MjVINDcwLjI1QzUyOS4yNzUgMzIwLjYyNSA1NzcuMTI1IDM2OC40NzUgNTc3LjEyNSA0MjcuNVY1NjIuODc1QzU3Ny4xMjUgNTc4LjYxNSA1NjQuMzY1IDU5MS4zNzUgNTQ4LjYyNSA1OTEuMzc1SDMwNi4zNzVDMjQ3LjM1IDU5MS4zNzUgMTk5LjUgNTQzLjUyNSAxOTkuNSA0ODQuNVYzNDkuMTI1WiIgZmlsbD0id2hpdGUiLz4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik02NzYuODE1IDU4My44MjlMNjE4LjI5MyA1MzguNjUzQzYxNC43OTcgNTM1Ljk1NSA2MTIuNzUgNTMxLjc4OSA2MTIuNzUgNTI3LjM3M1Y0MTMuMTI3QzYxMi43NSA0MDguNzExIDYxNC43OTcgNDA0LjU0NSA2MTguMjkzIDQwMS44NDdMNjc2LjgxNSAzNTYuNjcxQzY4MS4zNDUgMzUyLjAxNiA2ODcuNjc5IDM0OS4xMjUgNjk0LjY4OCAzNDkuMTI1QzcwOC40NiAzNDkuMTI1IDcxOS42MjUgMzYwLjI5IDcxOS42MjUgMzc0LjA2MlY1NjYuNDM4QzcxOS42MjUgNTgwLjIxIDcwOC40NiA1OTEuMzc1IDY5NC42ODggNTkxLjM3NUM2ODcuNjc5IDU5MS4zNzUgNjgxLjM0NSA1ODguNDg0IDY3Ni44MTUgNTgzLjgyOVoiIGZpbGw9IndoaXRlIi8+Cjwvc3ZnPgo=",
```

Comment:
It looks like there are some new entries here after running `make plugins.json`. @hanzei Any idea why this might be happening? For the purposes of this PR, I suggest we just update `repo_name` and the `platforms` properties and do not add any "missing" entries.

Reply:
I actually decided to include them, to avoid doing manual surgery on `plugins.json`

Reply:
The entries that were added:

- [gitlab 1.2.0](https://github.com/mattermost/mattermost-plugin-gitlab) (latest)
- [nps 1.0.2](https://github.com/mattermost/mattermost-plugin-nps) (not latest)
- [aws SNS 1.1.0](https://github.com/mattermost/mattermost-plugin-aws-SNS) (latest)
- [custom attributes 1.3.0](https://github.com/mattermost/mattermost-plugin-custom-attributes) (latest)
- [jira 3.0.0](https://github.com/mattermost/mattermost-plugin-jira) (latest)
- [autolink 1.2.1](https://github.com/mattermost/mattermost-plugin-autolink) (latest)
- [zoom 1.4.1](https://github.com/mattermost/mattermost-plugin-zoom) (latest)
- [zoom 1/4.1](https://github.com/mattermost/mattermost-plugin-zoom) (not latest)

Reply:
@jfrerich do you have any ideas why this may be happening when I run `make plugins.json`?

Reply:
I did the generate process over again, and made sure that any new unwanted entries were removed. I think this is fine for this PR, but I think we need to figure out why the new entries are being added.